docs/health-api-specs/contracts/registries/household.yml

```
@@ -2,785 +1,1169 @@
-          required: true
-          schema:
-            $ref: '#/definitions/HouseholdMemberSearchRequest'
-        - $ref: '#/parameters/limit'
-        - $ref: '#/parameters/offset'
-        - $ref: '#/parameters/tenantId'
-        - $ref: '#/parameters/lastChangedSince'
-        - $ref: '#/parameters/includeDeleted'
       tags:
         - Household Member
+      summary: Search existing household member
+      description:  Search existing household member
```

YAMLlint (1.35.1)

[error] too many spaces after colon

(colons)